game/scripts/vscripts/Sections/Chapter6/SectionClosing.ts

```diff
@@ -176,13 +188,67 @@ function onStart(complete: () => void) {
                 tg.audioDialog(LocalizationKey.Script_6_Closing_2, LocalizationKey.Script_6_Closing_2, sunsFan),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_3, LocalizationKey.Script_6_Closing_3, slacks),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_4, LocalizationKey.Script_6_Closing_4, sunsFan),
+
+                tg.withHighlights(
+                    tg.forkAny([
+                        tg.seq([
+                            tg.immediate(_ => {
+                                goalDestroyTowers.start()
+                                goalDestroyTowers.setValue(0)
+                                pathParticleID = createPathParticle([...pathLocations, ancient.GetAbsOrigin()])
+                            }),
+                            tg.completeOnCheck(_ => {
+                                towersToDestroy = towersToDestroy.filter(tower => unitIsValidAndAlive(tower))
+                                goalDestroyTowers.setValue(2 - towersToDestroy.length)
+
+                                return towersToDestroy.length === 0
+                            }, 0.1)
+                        ]),
+
+                        tg.seq([
+                            tg.panCameraExponential(getPlayerCameraLocation(), ancient.GetAbsOrigin(), 2),
+                            tg.audioDialog(LocalizationKey.Script_6_Closing_4, LocalizationKey.Script_6_Closing_4, sunsFan), // This needs to be edited with the new line for Sunsfan for destroying the ancient and the towers protecting it
+                            tg.panCameraExponential(ancient.GetAbsOrigin(), playerHero.GetAbsOrigin(), 2),
+                            tg.audioDialog(LocalizationKey.Script_6_Closing_4, LocalizationKey.Script_6_Closing_4, sunsFan), // This needs to be edited with the new line for Slacks for getting a Divine Rapier, and to destroy the ancient
+                            tg.immediate(() => playerHero.AddItemByName("item_rapier")),
+                            tg.immediate(() => {
+                                const tpScroll = playerHero.AddItemByName("item_tpscroll")
+                                Timers.CreateTimer(FrameTime(), () => {
+                                    tpScroll.EndCooldown()
+                                })
+                            }),
+                            tg.neverComplete()
+                        ])
+                    ]),
+                    {
+                        type: "arrow_enemy",
+                        units: towersToDestroy,
+                        attach: true
+                    }
+                ),
+
+                tg.withHighlights(
+                    tg.seq([
+                        tg.immediate(_ => {
+                            goalDestroyTowers.complete()
+                            goalDestroyAncient.start()
+                        }),
+                        tg.completeOnCheck(() => {
```

Comment:
Can make this 1 line `tg.completeOnCheck(_ => !unitIsValidAndAlive(ancient), 0.1),`

Reply:
Done.